### PR TITLE
[Waiting] AT/2320 Publish python to artifactory

### PR DESCRIPTION
**Pull request by dariuscox**
With this change we will now be pushing to artifactory using a .pypirc that exists in jenkins
When jenkins ami is updated, merge this

### CODE REVIEW

File: tasks/setup.rake [hunk 1]
```diff
@@ -53,6 +53,15 @@ namespace "setup" do
     version = Setup.get_package_version
     good("Published egg version #{version} to index #{Project[:INDEX_URL]}")
   end
+
+  desc "Publish the egg to the artifactory, make sure a there is a .pypirc"
+  task :artifactory => ["virtualenv:verify", "version:inject_git", "version:inject_build_number"] do
+    Dir.mktmpdir do |tmpdir|
+      Setup.setup %|sdist -d "#{tmpdir}" upload -r local|
```
Comment (aoskotsky-amplify) [inline tasks/setup.rake]: what command is this running? is this calling the jfrog cli?

Comment (dariuscox) [reply tasks/setup.rake]: This is running the setup.py script and is not using jfrog cli
i did it as suggested [here](https://www.jfrog.com/confluence/display/RTF/PyPI+Repositories) and tested it with astrojenkins

Comment (dariuscox) [reply tasks/setup.rake]: https://mhcastrojenkins-build.aws.wgen.net/jenkins/job/asiaq-BUILD/517/console

File: tasks/jenkins.rake [hunk 2]
```diff
@@ -230,10 +230,10 @@ namespace "jenkins" do
   end
 
   desc "Run a local build"
-  task :build => ["setup:install", :test, "setup:pynest", :component_rpmbuild, "doc:publish"]
```
Comment (dariuscox) [inline tasks/jenkins.rake]: possibly call both until the repo is shut down

**Review by jconstance-amplify — APPROVED** [review]

**Pull request merged**